plugins/bufferlines/navic.nix

```diff
@@ -0,0 +1,112 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.navic;
+  helpers = import ../helpers.nix {inherit lib;};
+  mkListStr = helpers.defaultNullOpts.mkNullable (types.listOf types.str);
+in {
+  options.plugins.navic =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "nvim-navic";
+
+      package = helpers.mkPackageOption "nvim-navic" pkgs.vimPlugins.nvim-navic;
+
+      icons =
+        mapAttrs
+        (name: default:
+          helpers.defaultNullOpts.mkStr default "icon for ${name}."
+        )
+        {
+          File = "󰈙 ";
+          ...
+        };
```

Comment:
Actually, you need to replace `...` with the other icons name-default pairs.

Reply:
Ahh I see :)

Reply:
fixed